crates/sui-graphql-rpc/src/lib.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-pub mod client;
+pub use sui_graphql_rpc_client as client;
```

Comment:
Do we still need this? It seems counter-intuitive to export the client from the server if we're splitting it out into its own crate.